meson.build

```diff
@@ -89,8 +89,24 @@ if openssl_dep.found()
            description: 'OpenSSL/LibreSSL API version @0@'.format(api_version))
 endif
 
-# Check for libdus availability. Optional, only required for MCTP dbus scan
-libdbus_dep = dependency('dbus-1', required: false)
+if get_option('libdbus').disabled()
+    libdbus_dep = dependency('', required: false)
```

Comment:
Could we use `declare_dependency` instead?

Reply:
Sounds like a good idea according the docs. We have some more of those. I add a separate commit to change all in one go

Reply:
Nope, doesn't work. The `declare_dependecy` dependency object will return true `.found()`.

Reply:
Good to know...

Reply:
And one thing I don't understand is that the documentation talks about internal build but we have to use declare_dependency to make the dependency being announced to be used in a wrap... I am too stupid too understand all these things :)

Reply:
I read the documentation and did not understand as well... :smirk: